crates/biome_css_analyze/tests/specs/nursery/noExcessiveSelectorClasses/valid.css

```
@@ -0,0 +1,5 @@
+/* should not generate diagnostics */
+div {}
```

Stylelint (17.6.0)

[error] Unexpected empty block (block-no-empty)

(block-no-empty)